### PR TITLE
Tweaks for Backstop tests

### PR DESCRIPTION
**Pull request by st3v3nhunt**
## Checklist

- [ ] An [ADR](http://thinkrelevance.com/blog/2011/11/15/documenting-architecture-decisions)
- [ ] New and/or updated tests
- [ ] Changes log in `CHANGELOG`

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -30,3 +30,5 @@ newrelic_agent.log
 backstop-tests/backstop_data/html_report/
 backstop-tests/backstop_data/bitmaps_test/
 backstop-tests/.idea
+backstop-tests/package.json
+backstop-tests/package-lock.json
```
Comment (Phil-87) [inline .gitignore]: I think package-lock.json should be committed, I don't know about package.json though. https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5

File: scripts/test-backstop [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+# Install mimimist for use in Backstop test scripts
+cd ./backstop-tests/ && npm init -f && npm install minimist@1.2.0
+docker run --rm -v "$(pwd)":/src backstopjs/backstopjs test --config=backstop.js
```
Comment (Phil-87) [inline scripts/test-backstop]: I had to add the "/" again, in front of the "$" to get this to work, probably a Windows issue. If this also works for you with "/$" I suggest that's how it should be.

**Comment by Phil-87** [comment]
Looks good, I added a couple of comments though. The tests pass when running locally in Docker, see the comment about "/$".

File: .gitignore (hunk 1 above)
Comment (st3v3nhunt) [reply .gitignore]: There isn't any need to do this as the package is installed as part of `./scripts/test-backstop`.

**Comment by Phil-87** [comment]
All looks good and still passing locally ✔️

File: scripts/README.md [hunk 3]
```diff
@@ -7,6 +7,10 @@ Below is a list of scripts available, along with a simple description of
 what each one does. The details of what they are doing is available within the
 script.
 
+[`approve-backstop`](approve-backstop)
+Approves changes found during the last run of Backstop stops. Backstop runs in
```
Comment (Phil-87) [inline scripts/README.md]: Did you mean "Backstop tests", not "Backstop stops"?

Comment (st3v3nhunt) [reply scripts/README.md]: Indeed I did. Good spot 👀

**Pull request merged**